### ISSUE TITLE
Lost event listeners

### ISSUE THREAD

**Issue by Fellach** [gmazzamuto/ng2-google-charts#26]
Hi,

first of all, thank you for porting the charts into angular's world.

I've got issue, when data are append after a chart creation. I'm using `Object.create()` in my `subscribe`'s callback to re-draw the chart with new options, what causes the method from `(chartSelect)` isn't called.

Am I doing wrong or is it a bug?

---
scrap of code for reference:
```js
export class Component {
    activityChartOptions: any;

    constructor(private service: Service) {
        this.activityChartOptions = require('./charts/activity.json');
    }

// ...

    private fetchStats() {
        this.service.get().subscribe((result) => {
            this.activityChartOptions = Object.create(this.activityChartOptions);
            this.activityChartOptions.dataTable = result;
        });
    }      
}
```

**Comment by gmazzamuto** [comment]
This looks ok, but you're not showing how you are binding to `chartSelect`.

**Comment by Fellach** [comment]
Thanks for the quick reply. Do you mean the html?

```
<div *ngIf="showCharts" >
   <google-chart [data]="activityChartOptions" (chartSelect)="onOpenDetails($event)"></google-chart>
</div>
```

This `showCharts: boolean` is a workaround. I set it to `false` in the `fetchStats()` before the service call 
 and then, in the callback, back to `true`. But that hides the chart for a moment.

**Comment by gmazzamuto** [comment]
I don't see anything wrong here. But again I can't see what type of chart you are trying to draw, what options you are passing from your json file, what the `onOpenDetails` function does.

In the demo there is an example where the `chartSelect` event is demonstrated. Can you please see if you can follow that example?

**Comment by Fellach** [comment]
HI again.

I took the code from your demo and modified the component a little to simplify and to simulate API call. As I've described above listener stopped working, when data are re-created. 

Cheers!

```js
export class AppComponent {

 public selectEvent: ChartSelectEvent;

 public lineChartData:any =  {
    chartType: 'LineChart',
    dataTable: [
      ['Year', 'Sales', 'Expenses'],
      ['Year', 0, 0],
    ],
    options: {title: 'Company Performance'}
  };

  ngOnInit() {
    setTimeout(() => { // eg. API call

      this.lineChartData = Object.assign({}, this.lineChartData, {
          dataTable: [
            ['Year', 'Sales', 'Expenses'],
            ['2004',  1000,      400],
            ['2005',  1170,      460],
            ['2006',  660,       1120],
            ['2007',  1030,      540]
          ]
      });

    }, 1000);
 }

  public select(event: ChartSelectEvent) {
    this.selectEvent = event;
  }
}
```

**Comment by viwell** [comment]
There is a bug when registering events on a loaded chart, I've created a pull request to fix it #31
